Sources/MapboxCoreNavigation/NavigationService.swift

```diff
@@ -148,12 +152,12 @@ public class MapboxNavigationService: NSObject, NavigationService {
     /**
      The active router. By default, a `RouteController`.
      */
-    public var router: Router!
+    public var router: Router { _router! }
```

Comment:
`router` is set after `super.init` so it is "safe" to make it non-optional. If it can crash it will crash in init method.

Reply:
I like this approach of keeping the implicitly unwrapped optional but making it an internal implementation detail instead of exposing it to developers.

Reply:
Can't agree more!